packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -26,6 +26,8 @@ class EnrollVerbHandler extends AbstractVerbHandler {
   @override
   Verb getVerb() => enrollVerb;
 
+  final _uuid = Uuid();
```

Comment:
I don't think this should be an instance variable.

Reply:
done.
https://github.com/atsign-foundation/at_server/pull/1496/commits/b22ea2028164aa54299758080d3b9033bf3a98e2